spec/loadable_spec.rb

```diff
@@ -12,7 +12,8 @@
 
   describe 'class methods' do
     describe '#load_validations' do
-      it 'returns the load_validations from the current and all ancestral classes in a hierarchical, defined order' do
+      it "returns the load_validations from the current \
```

Comment:
can we just slightly modify this to say `returns the load_validations from the current class`